docs/user/install-gke-celery.md

```diff
@@ -1,48 +1,71 @@
+locally running within the cluster.
 
-### **Prerequisites**
+### Prerequisites
 
-GKE is only supported for Google Cloud so a Google Cloud Project is required to work from.
+- A Google Cloud Account and a project to work from
+- The ability to create GCP resources and service accounts
+- `gcloud` and `kubectl` locally installed
 
-## **Installation**
+## Deployment
 
-Please follow these steps for deploying Turbinia to GKE. Ensure that the `.clusterconfig` config file has been updated appropriately.
+This section covers the steps for deploying a Turbinia GKE environment.
 
-### **Turbinia GKE Deployment**
-
-**Follow these steps to deploy Turbinia to GKE.**
+### Deploying the Turbinia cluster
 
 - Create or select a Google Cloud Platform project in the
   [Google Cloud Console](https://console.cloud.google.com).
 - Determine which GCP zone and region that you wish to deploy Turbinia into.
-- Review the `.clusterconfig` config file and please update any of the default values if necessary based on requirements.
-- Deploy Turbinia through the following command
+- Review the `.clusterconfig` config file and please update any of the default values if necessary based on cluster requirements.
+- Deploy through the following command:
   - `./k8s/tools/deploy-celery-gke.sh`
-- The deployment script will automatically enable GCP APIs, create the cluster and GCP resources then deploy Turbinia to the cluster. At the end of the run, you should have a fully functioning Turbinia environment within GKE to use.
+  - **Note this script will create a GKE cluster and GCP resources then deploy Turbinia to the cluster**
+- Congrats, you have successfully deployed Turbinia locally! Please see [install-gke-external](install-gke-external.md)
+  for instructions on accessing Turbinia externally through a URL or see the section below for making Turbinia processing requests locally.
 
-**Follow these steps to destroy the Turbinia GKE environment.**
+### Destroying the Turbinia cluster
 
-- Run the following command if you'd like to destroy the Turbinia GKE environment.
+- Run the following command if you'd like to destroy the Turbinia GKE environment:
   - `./k8s/tools/destroy-celery-gke.sh`
   - **Note this will delete the Turbinia cluster including all processed output and log files as well as associated GCP resources**
 
-### **Making processing requests in GKE**
+## Making requests locally
+
+If you have not set up external access to Turbinia, you can make a request through the following steps.
+
+- Connect to the cluster:
+
+```
+gcloud container clusters get-credentials <CLUSTER_NAME> --zone <ZONE> --project <PROJECT_NAME>
+```
+
+- Forward the Turbinia service port locally to your machine:
+
+```
+kubectl port-forward service/turbinia-api-service 8000:8000
+```
+
+- Please have the Turbinia client installed locally then create a processing request via:
+
+```
+turbinicatl googleclouddisk -d <DISK_NAME> -z <ZONE>
```

Comment:
s/turbinicatl/turbiniactl/

Will this actually work as is with turbiniactl?  I know later we'll use `turbiniamgmt` for this, but for now with turbiniactl AFAIK this would require access to Redis, but isn't the port forwarded above for the API server?  Will the config be set up correctly for this as well (I guess it would need to point to localhost)?